portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login-status")
+public class LoginStatusServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userLoggedIn;
+    response.setContentType("application/json");
+    PrintWriter out = response.getWriter();
+    UserService userService = UserServiceFactory.getUserService();
+    Gson gson = new Gson();
+
+    if (!userService.isUserLoggedIn()) {
+      userLoggedIn = gson.toJson("notLogged");
```

Comment:
You actually don't need to send anything in the body anymore.
I think you can just use the response status code to deduce.

Reply:
Yeah, I was actually wondering about this. However I read an article from https://www.javamex.com/tutorials/servlets/http_status_code.shtml that recommends we send something in the body to prevent some server info from being sent unintentionally. What do you think?

Reply:
I'm not familiar with that rule. Maybe you can give a try and observe whether a message body is automatically inserted?

Reply:
Removed it in commit f3148e2, I checked it out, I didn't notice any difference.

Reply:
Thanks for checking it out!